account_financial_risk_manager/models/res_partner.py

```diff
@@ -11,5 +11,5 @@ def _compute_risk_allow_edit(self):
         is_editable = self.env.user.has_group(
             "account_financial_risk_manager.group_risk_manager"
         )
-        for partner in self.filtered("customer"):
+        for partner in self.filtered(lambda x: x.customer_rank > 0):
```

Comment:
I wouldn't worry about this filter unless upstream module is doing the same. If all partners are potentially customers and customer rank is not set until their is an order, it would seem to break more than it fixes to filter here as we need to set limit before order.

As for rest of this, I need to wait till we see what really has replaced the view.